parsl/executors/high_throughput/mpi_prefix_composer.py

```diff
@@ -49,7 +31,7 @@ def validate_resource_spec(resource_spec: Dict[str, str]):
                       ))
     invalid_keys = user_keys - legal_keys
     if invalid_keys:
-        raise InvalidResourceSpecification(invalid_keys)
+        raise InvalidResourceSpecificationError(invalid_keys=invalid_keys)
```

Comment:
other places do not use keyword args `invalid_keys=` here, so it would be more consistent style to leave the parameter as a position argument